tests/functional/z_cluster/test_scale_mon_pods.py

```
@@ -0,0 +1,387 @@
+                    log.error("Failed to scale mon count back to 3 during teardown")
+                else:
+                    log.info("Successfully scaled mon count back to 3")
+        except Exception as e:
+            log.error(f"Error during mon count teardown: {e}")
+
+        try:
+            if self.are_rack_labels_present():
+                self.nodes = get_worker_nodes()
+                # We will scaledown the total racks back to three during teardown
+                self.racks = [
+                    "rack{}".format(i % (len(self.nodes) // 2))
```

Ruff (0.15.15)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)